app/components/hero/Hero.module.css

```diff
@@ -0,0 +1,20 @@
+.container {
+  position: absolute;
```

Comment:
Tohle bych urcite nedelal pomoci position absolute, to pak bude hrozna bolest vse poskladat dohromady. Pouzil bych normalne relative

Reply:
jo to je chyba samozdrejme